CODEOWNERS

```
@@ -0,0 +1,6 @@
+# These owners will be the default owners for everything in the repo.
+* @microsoft/dynamics-smb-engineering-systems @microsoft/dynamics-smb-developertools
```

Remove microsoft/dynamics-smb-developertools as it's not a public group.

```suggestion
* @microsoft/dynamics-smb-engineering-systems
```